src/chapters/strings/strings-as-objects.rst

```diff
@@ -0,0 +1,106 @@
+Strings as Objects
+==================
+
+Beyond bracket notation, there are many other tools we can use to work with strings. Talking about these tools requires some new terminology.
+
+Object Terminology
+------------------
+
+.. index:: object, property, method
+
+An **object** is collection of related data and operations. We have already encountered one object, the built-in object ``console``, which we use to output messages.
+
+.. admonition:: Example
+
+   .. sourcecode:: js
+
+      console.log(typeof console);
+
+   **Output**
+
+   ::
+   
+      object
+
+JavaScript reports that the type of ``console`` is indeed ``object``. An operation that can be carried out on an object is known as a **method**. 
+
+You can think of methods as functions that "belong to" an object. In other words, they only make sense when used in conjunction with an object. We can call ``console.log()``, but we may not use ``log()`` on its own.
+
+A piece of data that associated with an object is known as a **property**. While we have not had a reason to use them, the ``console`` object has ``name`` and ``length`` properties. 
```

Comment:
I see your point. I was trying to intro methods by talking about something they have seen. That gets a little sticky because `name` and `length` aren't properties of `console` that they will actually use. I'm open to a broader discussion on this.

Reply:
@welzie @jimflores5 Based on Blake's comment and talking with Jim, I have substantially revised this section. Give it a read, and let me know what you think.

Reply:
Minor typos aside, this section flows better than the previous console.length attempt.